### DISCUSSION TITLE
Working production example with ingress

### DISCUSSION THREAD

**Discussion by fabland** [keycloak/keycloak#12033] [Archive]
Would it be possible to have a working example of a recommended way of how to install Keycloak 18 through the new operator in a production set-up with an Ingress/SSL and a customized quarkus image? Depending on the combination of settings I am using for the Keycloak CRD different things are not working, e.g. some URLs are still referencing /auth on the keycloak pages, some settings seem to point to https://URL:8443 instead of https://URL and if I try to correct those with settings I have some problems with invalid redirect uris.

Either Ingress with pass-through and TLS termination on the keycloak pod or Ingress with re-encrypt would be great to see how it should work.

**Comment by vmuzikar** [comment]
Hello @fabland, thank you for your interest in the new Operator! I believe the issues you're facing are related to bugs we've found and we actually have a fix for: https://github.com/keycloak/keycloak/pull/12021

CC: @andreaTP

**Reply by andreaTP** [reply]
@vmuzikar already answered on the configuration/bugs side.

In general, having a full-blown "example" repository for the "production-ready" deployment we support seems like a good idea to me.
It might belong to the [quickstart guides](https://github.com/keycloak/keycloak-quickstarts)

cc. @DGuhr
